hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiringCreations.java

```diff
@@ -191,6 +191,14 @@ private ExpirableTxnRecord.Builder createBaseRecord(
 			baseRecord.setContractCreateResult(createResult);
 		}
 
+		if (sideEffectsTracker.hasTrackedRandomData()) {
+			if (sideEffectsTracker.getPseudorandomNumber() > 0) {
```

Comment:
As above, since `0` is a valid output, maybe we use `>= 0` here even if the protobuf serialization will skip it.

Reply:
Fixed !